apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/oms/OmstillingsstoenadInnvilgelse.kt

```diff
@@ -59,19 +66,17 @@ data class OmstillingsstoenadInnvilgelse(
                     OmstillingsstoenadBeregning(
                         innhold = innholdMedVedlegg.finnVedlegg(BrevVedleggKey.OMS_BEREGNING),
                         virkningsdato = avkortingsinfo.virkningsdato,
-                        inntekt = avkortingsinfo.inntekt,
-                        grunnbeloep = avkortingsinfo.grunnbeloep,
                         beregningsperioder = beregningsperioder,
-                        sisteBeregningsperiode = beregningsperioder.maxBy { it.datoFOM },
+                        sisteBeregningsperiode = sisteBeregningsperiode,
                         trygdetid =
                             TrygdetidMedBeregningsmetode(
                                 trygdetidsperioder = trygdetid.perioder,
                                 beregnetTrygdetidAar = trygdetid.aarTrygdetid,
                                 beregnetTrygdetidMaaneder = trygdetid.maanederTrygdetid,
                                 prorataBroek = trygdetid.prorataBroek,
                                 mindreEnnFireFemtedelerAvOpptjeningstiden = trygdetid.mindreEnnFireFemtedelerAvOpptjeningstiden,
-                                beregningsMetodeFraGrunnlag = avkortingsinfo.beregningsperioder.first().beregningsMetodeFraGrunnlag,
-                                beregningsMetodeAnvendt = avkortingsinfo.beregningsperioder.first().beregningsMetodeAnvendt,
+                                beregningsMetodeFraGrunnlag = sisteBeregningsperiode.beregningsMetodeFraGrunnlag,
+                                beregningsMetodeAnvendt = sisteBeregningsperiode.beregningsMetodeAnvendt,
```

Comment:
Hm, fra `first()` til `.maxBy { it.datoFom }`. Gir mening for meg, men her må noen med mer kjennskap til beregning si noe 🤔

Reply:
`first()` gjør en antagelse om at listen er sortert, noe man strengt tatt ikke bør ta for god fisk. Her gjør man en litt mer konservativ approach for å sikre at vi alltid får ut siste periode (dersom det skulle være flere).